pkg/framework/aries/framework.go

```diff
@@ -170,20 +170,13 @@ func WithKMS(k api.KMSCreator) Option {
 	}
 }
 
-// WithPacker injects a Packer service to the Aries framework
-// to pack outbound messages and be available for unpacking inbound messages.
-func WithPacker(c packer.Creator) Option {
+// WithPacker injects Packer services to the Aries framework.
+// The first parameter is used for outbound and inbound messages,
+// and the optional additional packers are for inbound messages.
+func WithPacker(c packer.Creator, inboundPackers ...packer.Creator) Option {
```

Comment:
we should remove inbound/outbound words from packer

c is the default/primary packer for packing/unpacking

inboundPackers is a list of packers to accommodate different types of envelopes.. yes, initially for inbound envelopes, but if in future communications 2 agents talk to each other with a different type than the primary one defined in `c`, then a packer from this list should be referenced.

this is why the word `inbound` can be misleading

Reply:
Good point, I'll call them primary and additional packers.